docs/dev/devel-env.rst

```diff
@@ -276,6 +276,13 @@ Also, run the **integration tests** for ListenBrainz.
 When the tests complete, you will see if your changes are valid or not. These tests
 are a helpful way to validate new changes without a lot of work.
 
+.. note::
```

Comment:
Putting this after tests doesn't make sense. Can we just put  it in the update config.py section?